src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -245,10 +245,13 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     "NavBarScheduleBreadcrumb__btn--disabled": !filterRelatedEvents,
   });
 
+  const selectedVenue =
+    (filterRelatedEvents ? venueId : sovereignVenue?.id) ?? "";
+
   return (
     <div className="NavBarWrapper">
       <div className={containerClasses}>
-        <NavBarScheduleWeather containerClassName="NavBarSchedule--end-to-end" />
+        {/* <NavBarScheduleWeather containerClassName="NavBarSchedule--end-to-end" /> */}
```

Comment:
why is that commented? @ermiaSparkle

Reply:
Looks like we are trying to solve too many things in one PR.

Reply:
In our morning session, we decided to comment on the weather until further notice.
(CC @sofisparkle )

Reply:
i see, thanks! i thought it was removed by mistake because there is no reference to that issue/request in your PR.